tket/src/Utils/MatrixAnalysis.cpp

```diff
@@ -525,13 +407,13 @@ get_information_content(const Eigen::Matrix4cd &X) {
   basis_change << 1, 1, -1, -1, -1, 1, -1, 1, 1, -1, -1, 1, 1, 1, 1,
       1;  // k3 = 0 always
   Eigen::Vector3d k =
-      (.25 * basis_change * thetas).head<3>().unaryExpr([](double d) {
-        return mod(d, 2 * PI);
+      (-.5 * basis_change * thetas / PI).head<3>().unaryExpr([](double d) {
+        return mod(d, 4);
       });
 
-  // move k into Weyl chamber ie pi/4 >= k_x >= k_y >= |k_z|
+  // move k into Weyl chamber ie 1/2 >= k_x >= k_y >= |k_z|
   //   1. permutate ks
-  //   2. modulo pi/2 and pi/4
+  //   2. modulo 1. and 1/2
   std::vector<int> ind_order{0, 1, 2};
```

Comment:
Not your code, but looks to me like `ind_order` could be a `std::array` instead

Reply:
Done.